tempesta_fw/http.h

```diff
@@ -232,6 +232,9 @@ typedef struct {
 #define TFW_HTTP_VOID_BODY		0x010000	/* Resp to HEAD req */
 #define TFW_HTTP_HAS_HDR_DATE		0x020000	/* Has Date: header */
 
+/* Used to indicate that @SS_SEND_F_PASS_SKB must not be used */
```

Comment:
An empty line before this one is confusing, as empty lines are used to separate blocks of definitions. Here is the block of definitions for responses. I would remove the empty line for that reason.